open_spiel/julia/wrapper/spieljl.cc

```diff
@@ -170,14 +173,16 @@ JLCXX_MODULE define_julia_module(jlcxx::Module& mod) {
           }
           return keys;
         });
-        wrapped.method("values", [](WrappedT ps) {
-          std::vector<WrappedVal> vals;
-          vals.reserve(ps.size());
-          for (auto const& it : ps) {
-            vals.emplace_back(it.second);
-          }
-          return vals;
-        });
+        // wrapped.method("values", [](WrappedT ps) {
+        //   std::vector<WrappedVal> vals;
+        //   vals.reserve(ps.size());
+        //   for (auto const& it : ps) {
+        //     vals.emplace_back(it.second);
+        //   }
+        //   return vals;
+        // });
```

Comment:
Should this remain here or be deleted?

Reply:
Deleted. Thanks. It used to be a workaround to resolve https://github.com/JuliaInterop/CxxWrap.jl/issues/190